src/test/java/com/uber/cadence/internal/replay/ReplayDeciderCacheTests.java

```diff
@@ -254,6 +254,7 @@ private ReplayDecider doNotCreateFakeDecider(
   }
 
   private ReplayDecider createFakeDecider(PollForDecisionTaskResponse response) {
+    SingleWorkerOptions options = mock(SingleWorkerOptions.class);
```

Comment:
This is causing issues for unit tests as SingleWorkerOptions is final and cannot be mocked. Just manually build a SingleWorkflowOptions instead.
```
com.uber.cadence.internal.replay.ReplayDeciderCacheTests > whenHistoryIsFullNewReplayDeciderIsReturnedAndCached_InitiallyEmpty FAILED

    org.mockito.exceptions.base.MockitoException: 

    Cannot mock/spy class com.uber.cadence.internal.worker.SingleWorkerOptions

    Mockito cannot mock/spy following:

      - final classes

      - anonymous classes

      - primitive types

        at com.uber.cadence.internal.replay.ReplayDeciderCacheTests.createFakeDecider(ReplayDeciderCacheTests.java:257)

        at com.uber.cadence.internal.replay.ReplayDeciderCacheTests.lambda$whenHistoryIsFullNewReplayDeciderIsReturnedAndCached_InitiallyEmpty$0(ReplayDeciderCacheTests.java:64)

        at com.uber.cadence.internal.replay.DeciderCache.getOrCreate(DeciderCache.java:65)

        at com.uber.cadence.internal.replay.ReplayDeciderCacheTests.whenHistoryIsFullNewReplayDeciderIsReturnedAndCached_InitiallyEmpty(ReplayDeciderCacheTests.java:64)
```

Reply:
Done